qserv-deploy.sh

```diff
@@ -16,21 +16,23 @@ usage() {
 Usage: `basename $0` [options] [cmd]
 
   Available options:
+    -C          Path to configuration directory (default to QSERV_CFG_DIR)
     -d          run in development mode (i.e. mount source files on host)
     -h          this message
 
   Run a docker container with all the Qserv deployment tools inside.
 
-  Pre-requisites: QSERV_CFG_DIR env variable must be defined and exported.
+  Pre-requisites: QSERV_CFG_DIR env variable can be defined and exported.
 
 EOD
 }
 
 MOUNT_SSH=true
 
 # get the options
-while getopts dhs c ; do
+while getopts C:dhs c ; do
```

Comment:
I don't see option`-s` explained in the "Usage" section above. Could it be fixed please?